extension/fluentbitextension/config.go

```diff
@@ -0,0 +1,43 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fluentbitextension
+
+import (
+	"go.opentelemetry.io/collector/config/configmodels"
+)
+
+// Config has the configuration for the fluentbit extension.
+type Config struct {
+	configmodels.ExtensionSettings `mapstructure:",squash"`
+
+	// The TCP `host:port` to which the subprocess should send log entries.
+	// One of this or `unix_socket_path` are required.
+	TCPEndpoint string `mapstructure:"tcp_endpoint"`
+
+	// The path to the executable for FluentBit
+	ExecutablePath string `mapstructure:"executable_path"`
```

Comment:
Is this absolute or relative?

Reply:
Can be both but preferably absolute since anything in the collector could do a change dir and the extension can't really prevent it.

Reply:
Also if you just put a single element name with no path separators, the Go exec library will look up the binary according to the PATH envvar set on the collector.